draft-ietf-httpbis-message-signatures.md

```diff
@@ -1537,20 +1546,54 @@ Specification document(s):
 
-|Name|Status|Target|Specification document(s)|
-|--- |--- |--- |--- |
-|`@signature-params`| Active | Request, Response | {{signature-params}} of this document|
-|`@method`| Active | Request | {{content-request-method}} of this document|
-|`@authority`| Active | Request | {{content-request-authority}} of this document|
-|`@scheme`| Active | Request | {{content-request-scheme}} of this document|
-|`@target-uri`| Active | Request | {{content-target-uri}} of this document|
-|`@request-target`| Active | Request | {{content-request-target}} of this document|
-|`@path`| Active | Request | {{content-request-path}} of this document|
-|`@query`| Active | Request | {{content-request-query}} of this document|
-|`@query-param`| Active | Request | {{content-request-query-param}} of this document|
-|`@status`| Active | Response | {{content-status-code}} of this document|
+|Name|Description|Status|Target|Specification document(s)|
+|--- |--- |--- |--- |--- |
+|`@signature-params`| Signature parameters, including covered content list | Active | Request, Response | {{signature-params}} of {{&SELF}}|
+|`@method`| The HTTP request method | Active | Request | {{content-request-method}} of {{&SELF}}|
+|`@authority`| The HTTP authority, or target host | Active | Request | {{content-request-authority}} of {{&SELF}}|
+|`@scheme`| The URI scheme of the request URI | Active | Request | {{content-request-scheme}} of {{&SELF}}|
+|`@target-uri`| The full target URI of the request | Active | Request | {{content-target-uri}} of {{&SELF}}|
+|`@request-target`| The request target of the request | Active | Request | {{content-request-target}} of {{&SELF}}|
+|`@path`| The full path of the request URI | Active | Request | {{content-request-path}} of {{&SELF}}|
+|`@query`| The full query of the request URI | Active | Request | {{content-request-query}} of {{&SELF}}|
+|`@query-param`| | A single named query parameter | Active | Request | {{content-request-query-param}} of {{&SELF}}|
+|`@status`| The status code of the response | Active | Response | {{content-status-code}} of {{&SELF}}|
 {: title="Initial contents of the HTTP Signature Derived Component Names Registry." }
 
+## HTTP Signature Component Parameters Registry {#component-param-registry}
+
+This document defines a component identifiers, which can be parameterized in specific circumstances to provide unique modified behavior. IANA is asked to create and maintain a new registry typed "HTTP Signature Component Parameters" to record and maintain the set of non-field component names and the methods to produce their associated component values. Initial values for this registry are given in {{iana-component-param-contents}}.  Future assignments and modifications to existing assignments are to be made through the Expert Review registration policy {{?RFC8126}} and shall follow the template presented in {{iana-component-param-template}}.
```

Comment:
```suggestion
This document defines component identifiers, which can be parameterized in specific circumstances to provide unique modified behavior. IANA is asked to create and maintain a new registry typed "HTTP Signature Component Parameters" to record and maintain the set of parameters for component identifiers, and the methods to produce their associated parameter values. Initial values for this registry are given in {{iana-component-param-contents}}.  Future assignments and modifications to existing assignments are to be made through the Expert Review registration policy {{?RFC8126}} and shall follow the template presented in {{iana-component-param-template}}.
```

Reply:
Ugh, copy-paste mistakes here. Changing it to this instead:

```suggestion
This document defines several kinds of component identifiers, some of which can be parameterized in specific circumstances to provide unique modified behavior. IANA is asked to create and maintain a new registry typed "HTTP Signature Component Parameters" to record and maintain the set of parameters names, the component identifiers they are associated with, and the modifications these parameters make to the component value. Initial values for this registry are given in {{iana-component-param-contents}}.  Future assignments and modifications to existing assignments are to be made through the Expert Review registration policy {{?RFC8126}} and shall follow the template presented in {{iana-component-param-template}}.
```